src/calculateRentalCost.js

```diff
@@ -3,8 +3,26 @@
  *
  * @return {number}
  */
+const PRICE_FOR_DAY = 40;
+const SHORT_TERM = 3;
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+const SHORT_TERM_DISCOUNT = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  let res = 0;
+
+  for (let i = 0; i < days; i++) {
```

Comment:
Using a loop to calculate the total price is unnecessary and inefficient. You can calculate the total cost by multiplying the `PRICE_FOR_DAY` by the number of `days` directly.